docs/cell_census_schema_0.0.1.md

```diff
@@ -0,0 +1,853 @@
+
+#### Cell Census information `census_obj[“census_info”]` - `SOMACollection`
+
+A series of summary and metadata tables MUST be included in this `SOMACollection`:
+
+##### Cell Census metadata – `census_obj​​[“census_info”][“summary”]` – `SOMADataFrame`
+
+Census metadata MUST be stored as a `SOMADataFrame` with two columns:
+
+<table>
+<thead>
+  <tr>
+    <th>Column</th>
+    <th>Encoding</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>label</td>
+    <td>string</td>
+    <td>Human readable label of metadata variable</td>
+  </tr>
+  <tr>
+    <td>value </td>
+    <td>string</td>
+    <td>Value associated to metadata variable</td>
+  </tr>
+</tbody>
+</table>
+
+This `SOMADataFrame` MUST have the following rows::
+ 
+1. Cell Census schema version:
+	1. label: `“cell_census_schema_version”`
+	1. value: Semver schema version.
+1. Cell Census build date:
+	1. label: `“cell_census_build_date”`
+	1. value: The date this Cell Census was built in ISO 8601 date format
+1. Total number of cells included in this Cell Census build:
+	1. label: `“total_cell_count”`
+	1. value: Cell count
+1. Unique number of cells included in this Cell Census build (is_primary_data == True)
+	1. label: `“unique_cell_count”`
+	1. value: Cell count
+1. Number of human donors included in this Cell Census build. Donors are guaranteed to be unique within datasets, not across all Cell Census.
+	1. label: `“number_donors_homo_sapiens”`
+	1. value: Donor count
+1. Number of mouse donors included in this Cell Census build. Donors are guaranteed to be unique within datasets, not across all Cell Census.
+	1. label: `“number_donors_mus_musculus”`
+	1. value: Donor count
+
+An example of this `SOMADataFrame` is shown below:
+
+<table>
+<thead>
+  <tr>
+    <th>label</th>
+    <th>value</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>cell_census_schema_version</td>
+    <td>0.0.1</td>
+  </tr>
+  <tr>
+    <td>cell_census_build_date</td>
+    <td>2022-11-30</td>
+  </tr>
+  <tr>
+    <td>total_cell_count</td>
+    <td>10000</td>
+  </tr>
+  <tr>
+    <td>unique_cell_count</td>
+    <td>1000</td>
+  </tr>
+  <tr>
+    <td>number_donors_homo_sapiens</td>
+    <td>100</td>
+  </tr>
+  <tr>
+    <td>number_donors_mus_musculus</td>
+    <td>100</td>
+  </tr>
+</tbody>
+</table>
+
+#### Cell Census table of CELLxGENE Discover datasets – `census_obj[“census_info”][“datasets”]` – `SOMADataFrame`
+
+All datasets used to build the Cell Census MUST be included in a table modeled as a `SOMADataFrame`. Each row MUST correspond to an individual dataset with the following columns:
+
+<table>
+<thead>
+  <tr>
+    <th>Column</th>
+    <th>Encoding</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>collection_id</td>
+    <td>string</td>
+    <td rowspan="5">As Defined in CELLxGENE Discover <a href="https://github.com/chanzuckerberg/single-cell/blob/main/design-documents/0001-data-portal-architecture/text.md#data-model">data schema</a>.</td>
```

Comment:
Are there plans to update the data model doc? Or even just the linked section/chart?

Reply:
yup

https://github.com/chanzuckerberg/cell-census/issues/21